clients/devtools-entry.js

```diff
@@ -14,13 +14,15 @@ const {registerLocaleData, lookupLocale} = require('../lighthouse-core/lib/i18n/
 
 /**
  * Return a version of the default config, filtered to only run the specified
- * categories.
+ * categories. Exclude `lighthouse-plugin-publisher-ads` to not include the
```

Comment:
so pub ads runs even if `categoryIDs` doesn't include the plugin category?

Reply:
I'll reword that to be less confusing :) it's trying to say it's the same as the others... It'll run if it's in categoryIDs, otherwise no

Reply:
+1 to a reword :)

we should also add a test for the `onlyCategories`/`plugins` interaction, it seems non-obvious to me what the behavior should be I never really thought about it.